test/server/cards/02_SHD/units/OmegaPartOfTheSquad.spec.ts

```diff
@@ -0,0 +1,65 @@
+describe('Omega, Part of the Squad', function() {
+    integration(function(contextRef) {
+        describe('Omega\'s Ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['omega#part-of-the-squad'],
+                        deck: ['crosshair#following-orders', 'pyke-sentinel', 'atst', 'cartel-spacer', 'battlefield-marine'],
+                        base: 'echo-base',
+                        leader: 'hera-syndulla#spectre-two'
+                    }
+                });
+            });
+
+            it('can draw a clone', function () {
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.omega);
+                expect(context.player1).toHavePrompt('Select a card to reveal');
+                expect(context.player1).toHaveEnabledPromptButton(context.crosshair.title);
+                expect(context.player1).toHaveDisabledPromptButtons([context.atst.title, context.battlefieldMarine.title, context.cartelSpacer.title, context.pykeSentinel.title]);
+                context.player1.clickPrompt(context.crosshair.title);
+                expect(context.crosshair).toBeInLocation('hand');
+                expect(context.getChatLogs(2)).toContain('player1 takes Crosshair');
+
+                // Omega should cost 4 since it cannot discount itself
+                expect(context.player1.countExhaustedResources()).toBe(4);
+
+                context.player2.passAction();
+
+                // Crosshair should cost 6 since Omega was the first Clone played this round
+                context.player1.clickCard(context.crosshair);
+                expect(context.player1.countExhaustedResources()).toBe(10);
+            });
+        });
+
+        describe('Omega\'s Ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['omega#part-of-the-squad'],
+                        hand: ['crosshair#following-orders', 'wolffe#suspicious-veteran'],
+                        base: 'echo-base',
+                        leader: 'hera-syndulla#spectre-two'
+                    },
+                });
+            });
+
+            it('negates aspect penalties on the first clone played', function () {
```

Comment:
If we have this test case can we just move all of the aspect penalty stuff from the first one to here? Right now it's kind of spread out between both

Reply:
I was trying to consolidate into two tests instead of three. I combined the play Omega test with the "don't discount a clone played right after Omega" test. I wouldn't be able to verify that in this test since Omega is already on the board

Reply:
Oh yeah that's fair, I didn't realize you wanted to make a distinction between the two. That's reasonable though. If you wanted you could combine them down to one using multiple action phases but I'm fine either way